src/mds/OpenFileTable.cc

```diff
@@ -31,6 +31,24 @@ static ostream& _prefix(std::ostream *_dout, MDSRank *mds) {
   return *_dout << "mds." << mds->get_nodeid() << ".openfiles ";
 }
 
+OpenFileTable::OpenFileTable(MDSRank *m) : mds(m) {
+  stringstream ss_name;
+  ss_name << "mds." << m->get_nodeid() << "-open_files_table";
+
+  PerfCountersBuilder b(mds->cct, ss_name.str(), l_oft_first, l_oft_last);
+
+  b.add_u64(l_oft_omap_total_objs, "omap_total_objs");
+  b.add_u64(l_oft_omap_total_kv_pairs, "omap_total_kv_pairs");
+  b.add_u64(l_oft_omap_total_updates, "omap_total_updates");
+  b.add_u64(l_oft_omap_total_removes, "omap_total_removes");
+  logger = b.create_perf_counters();
+  mds->cct->get_perfcounters_collection()->add(logger);
```

Comment:
Destructor for `OpenFileTable` must also remove the logger.

Reply:
done